File: Client/Zaposleni.js
export class Zaposleni
{
    constructor(ime, prezime, adresa, jmbg){
        this.ime = ime;
        this.prezime = prezime;
        this.adresa = adresa;
        this.jmbg = jmbg;
    }

    crtajZaposlene(host, i){

        var tr = document.createElement("tr");
        host.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML = this.ime;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML = this.prezime;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML = this.adresa;
        tr.appendChild(el);

        el = document.createElement("td");
        el.innerHTML = this.jmbg;
        tr.appendChild(el);
    }
}

File: Models/Zaposleni.cs
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;
using System;
using System.Text.Json.Serialization;

namespace Models
{
    public class Zaposleni
    {
        [Key]
        public int ID{get; set;}
        [Required]
        [MaxLength(50)]
        public string ime{get; set;}
        [Required]
        [MaxLength(50)]
        public string prezime{get; set;}
        [Required]
        [MaxLength(50)]
        public string adresa{get; set;}
        [Required]
        [RegularExpression("^[0-9]*$")]
        public long jmbg{get; set;}
        public Azil Azil{get; set;}
        public List<Zivotinja> Zivotinje{get; set;}
        
    }
}
  